src/Components/NotFoundErrorPage.tsx

```diff
@@ -1,12 +1,30 @@
 import { useEffect } from 'react'
-import { connect, ContentTag, Obj } from 'scrivito'
+import {
+  connect,
+  ContentTag,
+  ensureUserIsLoggedIn,
+  isUserLoggedIn,
+  load,
+  Obj,
+  NotFoundErrorPage as ScrivitoNotFoundErrorPage,
+} from 'scrivito'
+import { portalSiteAvailable } from '../config/scrivitoSites'
 
 // Make sure, that you have a proxy running for these URLs, otherwise you'll see an endless loop.
 const RELOAD_SUBPATHS = ['/auth']
 
 export const NotFoundErrorPage = connect(function NotFoundErrorPage() {
-  const root = Obj.root()
+  return (
+    <>
+      <ScrivitoNotFoundErrorPage>
+        <NotFound />
+      </ScrivitoNotFoundErrorPage>
+      <PortalSiteNotFound />
```

Comment:
* [ ] FYI We could make the fact we're only rendering one of these two clearer by conditional rendering, e.g.:
```tsx
  return currentSiteId() === null ? (
    <PortalSiteNotFound />
  ) : (
    <ScrivitoNotFoundErrorPage>
      <NotFound />
    </ScrivitoNotFoundErrorPage>
  )
```